packages/clipboard/README.md

```diff
@@ -21,27 +21,119 @@ yarn add @solid-primitives/clipboard
 
 ## How to use it
 
+### makeClipboard
+
+A basic non-reactive primitive that makes accessing the clipboard easy. Note that write supports both string and ClipboardItems object structure.
+
+```ts
+const [write, read, newItem] = makeClipboard();
+```
+
+#### Definition
+
+```ts
+function makeClipboard(): [
+  write: ClipboardSetter,
+  read: () => Promise<ClipboardItems | undefined>,
+  newItem: NewClipboardItem
+];
+```
+
 ### createClipboard
 
-Clipboard exports a read and write function. Note the write function is exported first for convenience as the most common use case for this primitive.
+This primitive provides full facilities for reading and writing to the clipboard. It allows for writing to clipboard via exported function or input signal. It wraps the Clipboard Async API with a resource and supplies reactive helpers to make pulling from the clipboard easy.
+
+```tsx
+const [data, setData] = createSignal('Hello);
```

Comment:
Lol must have missed the notification. ehh
Found a typo though. The missing `'` is messing the code highlighting.